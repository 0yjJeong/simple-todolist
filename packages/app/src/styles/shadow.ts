const shadow = {
  shadow0: '0 4px 9px 0 rgb(0 0 0 / 7%)',
};

export default shadow;
