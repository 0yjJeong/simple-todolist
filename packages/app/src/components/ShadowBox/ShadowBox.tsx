import styled from 'styled-components';
import shadow from '../../styles/shadow';

const ShadowBox = styled.div`
  background: #ffffff;
  padding: 0.8rem;
  border-radius: 10px;
  height: calc(100% - 100px);
  box-shadow: ${shadow.shadow0};
`;

export default ShadowBox;
