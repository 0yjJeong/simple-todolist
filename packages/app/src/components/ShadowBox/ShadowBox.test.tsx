import { render } from '@testing-library/react';
import 'jest-styled-components';
import ShadowBox from './ShadowBox';
import shadow from '../../styles/shadow';

describe('<ShadowBox/>', () => {
  it('works', () => {
    const rendered = render(<ShadowBox />);
    expect(rendered.container.firstChild).toMatchSnapshot();
    expect(rendered.container.firstChild).toHaveStyleRule(
      'box-shadow',
      shadow.shadow0
    );
  });
});
