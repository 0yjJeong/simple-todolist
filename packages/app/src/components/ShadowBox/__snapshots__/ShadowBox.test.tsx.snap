// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ShadowBox/> works 1`] = `
.c0 {
  background: #ffffff;
  padding: 0.8rem;
  border-radius: 10px;
  box-shadow: 0 4px 9px 0 rgb(0 0 0 / 7%);
}

<div
  class="c0"
/>
`;
